internal/worker/sshsession/worker.go

```diff
@@ -0,0 +1,349 @@
+package sshsession
+
+import (
+	"bufio"
+	"io"
+	"net"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/juju/errors"
+	jujussh "github.com/juju/utils/v3/ssh"
+	"github.com/juju/worker/v3"
+	"github.com/juju/worker/v3/catacomb"
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/juju/juju/core/watcher"
+	"github.com/juju/juju/rpc/params"
+)
+
+var (
+	// ControllerSSHUser is the user that will connect to the controller's embedded
+	// SSH server.
+	ControllerSSHUser = "reverse-ssh"
```

Comment:
Should we define this variable in one place and re-use it? I've updated it in my tunneler PR to `juju-reverse-tunnel` or soemthing like that based on a suggestion from Harry so it would be nice if they always stay in sync.

Reply:
Once both are merged, let's make a variable they can share then - agreed.